collect_app/src/main/java/org/odk/collect/android/geo/GoogleMapFragment.java

```diff
@@ -160,11 +167,21 @@ public class GoogleMapFragment extends SupportMapFragment implements
     @Override public void onAttach(@NonNull Context context) {
         super.onAttach(context);
         DaggerUtils.getComponent(context).inject(this);
+
+        GoogleMapConfigurator configurator = new GoogleMapConfigurator(
```

Comment:
We construct `GoogleMapConfigurator` in two places in the same way. Maybe those parameters should be a part of that class and we do not need to pass them?

Reply:
It would be replaced with an `@Inject MapConfigurator mapConfigurator` like in `OsmDroidMapFragment` but as in https://github.com/getodk/collect/pull/5139#discussion_r876203520, I'm can avoid that entirely and use `FragmentFactory` in my follow-up PR.